inferentia/scripts/setup.sh

```diff
@@ -115,53 +131,49 @@ apt-get update && \
               libarchive-dev   \
               rapidjson-dev
 
-# Using CMAKE installation instruction from:: https://apt.kitware.com/
-apt install -y gpg wget && \
-    wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | \
-        gpg --dearmor - |  \
-        tee /usr/share/keyrings/kitware-archive-keyring.gpg >/dev/null && \
-    . /etc/os-release && \
-    echo "deb [signed-by=/usr/share/keyrings/kitware-archive-keyring.gpg] https://apt.kitware.com/ubuntu/ $UBUNTU_CODENAME main" | \
-    tee /etc/apt/sources.list.d/kitware.list >/dev/null && \
-    apt-get update && \
-    apt-get install -y --no-install-recommends cmake cmake-data
-cmake --version
-
-# First compile correct python stub
-cd ${PYTHON_BACKEND_PATH}
-rm -rf build && mkdir build && cd build
-cmake -DTRITON_ENABLE_GPU=ON -DCMAKE_INSTALL_PREFIX:PATH=`pwd`/install ..
-make triton-python-backend-stub -j16
 
 # Set Pip repository  to point to the Neuron repository
 # since we need to use pip to update: 
 #  https://aws.amazon.com/blogs/developer/neuron-conda-packages-eol/
 pip config set global.extra-index-url https://pip.repos.neuron.amazonaws.com
+pip install --upgrade pip
+
+if [ ${INSTALL_INF2} -eq 1 ] || [ ${INSTALL_TRN1} -eq 1];then
+    # Install Neuron Runtime 
+    # Then install new neuron libraries
+    . /etc/os-release
+    tee /etc/apt/sources.list.d/neuron.list > /dev/null <<EOF
+deb https://apt.repos.neuron.amazonaws.com ${VERSION_CODENAME} main
+EOF
+    wget -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB |  apt-key add -
+    apt-get update
+    apt-get install -y aws-neuronx-collectives=2.* aws-neuronx-runtime-lib=2.*
+fi
 
-python -m pip install -U pip
-# Set pip repository pointing to the Neuron repository 
-python -m pip config set global.extra-index-url https://pip.repos.neuron.amazonaws.com
-if [ $USE_TENSORFLOW -eq 1 ]; then
+
+if [ ${USE_TENSORFLOW} -eq 1 ]; then
+    # conda install tensorflow-neuron pillow -y
     # Update Neuron TensorFlow
-    if [ $TENSORFLOW_VERSION -eq 1 ]; then
-        # Install TensorFlow Neuron
-        python -m pip install tensorflow-neuron[cc]==1.15.5.* "protobuf"
-    else
-        # Install TensorFlow Neuron
-        python -m pip install tensorflow-neuron[cc] "protobuf"
+    if [ ${INSTALL_INF1} -eq 1] && [ ${TENSORFLOW_VERSION} -eq 1 ]; then
+        pip install --upgrade tensorflow-neuron==1.15.5.* neuron-cc "protobuf<4" tensorboard-plugin-neuron
+    elif [ ${INSTALL_INF1} -eq 1]; then
+        pip install --upgrade tensorflow-neuron[cc] "protobuf<4"
```

Comment:
@nskool do you know why `tensorboard-plugin-neuron` is not installed if inf1 and TF2 is set? And why `protobuf<4` is specified?

I am validating the case when inf1 and tf1 is set and there is dependency w.r.t. protobuf version, grpc dependency seems to be coming from `tensorboard-plugin-neuron` (`Requirement already satisfied: grpcio>=1.48.2 in /usr/local/lib/python3.8/dist-packages (from tensorboard>=1.15.0->tensorboard-plugin-neuron) (1.54.0)`)
```
grpcio-channelz 1.54.0 requires protobuf>=4.21.6, but you have protobuf 3.20.1 which is incompatible.
```

Reply:
some update, looks like TF1 has shipped with protobuf 3.x dependency. I wonder if `tensorboard-plugin-neuron` is required or can it be pinned to a version works with protobuf 3.x if exists such an version.

Reply:
@GuanLuo thanks - I see that `tensorboard-plugin-neuron` was removed in this update - https://github.com/triton-inference-server/python_backend/pull/236/files. Do you happen to know the reason for that? It seems I can remove tensorboard plugin altogether as per the neuron documentation and also unpin the protobuf version - https://awsdocs-neuron.readthedocs-hosted.com/en/latest/general/quick-start/torch-neuron.html

I'm unaware of the protobuf dependency, but have chosen to retain the previous default configuration.

Reply:
The removal is probably unintended, however, from the [original discussion](https://github.com/triton-inference-server/python_backend/pull/92/files#r743935898) of adding this dependency, it was added just because it is being mentioned in the quickstart guide, i.e.:
```
# Install Neuron TensorBoard
python -m pip install tensorboard-plugin-neuron
```
Looking at the use of TensorBoard as a plugin, I think it is only used for further profiling the model and not necessary for running inference on Inferentia (current test runs fine without it). So I think we can remove the TensorBoard dependency to simplify the script if you agree that should not have significant impact.

Reply:
@GuanLuo I rebased with mainline, and added tensorboard-plugin for both TF1 and TF2 as per neuron instructions. That is correct, it is only for additional profiling. I have unpinned the protobuf dependency, and updated the readme as well.